plugins.json

```diff
@@ -802,5 +802,17 @@
         "download": "https://github.com/hernandezfo/AttachDocumentsforViewers/releases/download/0.0.1/AttachDocumentsforViewers-0.0.1.zip",
         "remote_install": true,
         "compatible_version": ">=1.2.5"
+    },
+    "zulip": {
+        "title": "Zulip",
+        "version": "1.1.0",
+        "author": "Peter Fejer",
+        "license": "MIT",
+        "description": "Receive individual or project notifications on Zulip.",
+        "homepage": "https://github.com/ptr0x01/kanboard-plugin-zulip",
+        "readme": "https://github.com/ptr0x01/kanboard-plugin-zulip/blob/master/README.md",
+        "download": "https://github.com/ptr0x01/kanboard-plugin-zulip/releases/download/1.1.0/Zulip.zip",
```

Comment:
When extracting the archive, the parent folder should be "Zulip" with all files inside.

Reply:
Ah sorry I didn't catch that, I've updated the release version it now contains the correct folder structure. Thanks!